### DISCUSSION TITLE
Migration story from gatsby -> hugo -> mkdocs-material

### DISCUSSION THREAD

**Discussion by brabster** [squidfunk/mkdocs-material#6997] [Show and tell]
Just a blog post on my new mkdocs-material site... https://tempered.works/posts/2024/03/29/now-on-mkdocs-material/

**Comment by squidfunk** [comment]
Awesome read, thanks for sharing! I always love to read _why_ users switch from other solutions to Material for MkDocs and where we can improve. Some notes:

> There's an interesting aspect there. mkdocs-material provides its own plugins for some optional features and mentions (recommends?) specific third-party plugins (like the glightbox or RSS plugins) for others. It feels like the author is taking some responsibility for the availability and safety of those third-party plugins. I'm not sure whether this is really the case, maybe I'll shoot a query over.

The built-in plugins try to be as complementary and useful as possible – we try to create plugins specifically for things that might need theme support and that are useful to our many users. Think blog, social cards, optimization, privacy – all of those try to solve pretty big problems well and work with each other. For instance, you can configure the privacy plugin to download specific resources from third-party locations and optimize images downloaded with the optimize plugin, e.g. screenshots that you use in your documentation and that your teams stores in a location outside of your repository. However, we also try to integrate well with third-party plugins that we view as useful or that have a wide adoption in the MkDocs ecosystem. After all, we want the ecosystem to prosper – it is awesome!

> I'm very happy to sign up for $15 per month to support this brilliant effort. The project is currently funded to the tune of nearly $15k/month! I appreciate the effort Martin has put into explaining the funding and support model, and it gives me the confidence to hand over my cash - in exchange for a supported product, not for someone to pursue their hobby project.

Thanks! Love to read this feedback on our Insiders pages – @katharinalisalin is currently reworking them to be even better and explain the ins and outs of the program. It is a hell of an experiment

**Reply by squidfunk** [reply]
Also note that most of the plugins work outside of Material for MkDocs with other MkDocs themes! However, it makes no sense for us to move them outside, since we want to provide a single awesome experience to our sponsors.
